hw/system/occamy/src/occamy_quadrant_s1_ctrl/occamy_quadrant_s1_reg.hjson

```diff
@@ -24,9 +24,9 @@
       desc: "Quadrant-internal asynchronous active-low reset",
       swaccess: "rw",
       hwaccess: "hro",
-      // Held in reset (i.e. signal high) by default
+      // Held in reset (i.e. signal low) by default
       fields: [
-        {bits: "0:0", name: "reset_n", resval: 1, desc: "Asynchronous active-low reset"}
```

Comment:
Agree?

Reply:
We discussed this today; for now yes. There are benefits and drawbacks to both, but this is a probably safer default since it forces the quadrants into a defined state on global reset.